contracts/modules/SSVDAO.sol

```diff
@@ -15,6 +15,8 @@ contract SSVDAO is ISSVDAO {
     uint64 private constant MINIMAL_LIQUIDATION_THRESHOLD = 100_800;
 
     function updateNetworkFee(uint256 fee) external override {
+        if (fee == 0) revert FeeTooLow();
```

Comment:
I don't think we must restrict the networkFee to be greater than 0.
Maybe the DAO wants it temporarily for some reason. @andrew-blox thoughts?